pymatgen/analysis/phase_diagram.py

```diff
@@ -74,42 +75,22 @@ def __init__(self, composition: Composition, energy: float,
                 analysis and plotting purposes. An attribute can be anything
                 but must be MSONable.
```

Comment:
It looks like this part of the docstring is supposed to go underneath the `attribute` attribute and not in the init

Reply:
I think you can delete the "analysis and plotting purposes"